docs-site/docs/configuration/pipelines.md

```diff
@@ -12,30 +12,47 @@ Currently only **Nextflow** pipelines are supported and they must have a Github
 
 ## Setup
 
+### Configuration
+
 The configuration file to register pipelines is in `json` format and stored in the `config/pipelines/` directory with the name `pipelines.json`.
 
-This `pipelines.json` file should be in the format below and must include the following:
+This `pipelines.json` file should be in the format below and can include the following:
+
+- **URL** *(Required)* of the pipeline Github repository
```

Comment:
Ensure GitHub is captialized properly.

Reply:
Fixed in https://github.com/phac-nml/irida-next/pull/780/commits/565c1987567bb3b93aa8d6119602c53c7d6764ed